apps/files/css/files.scss

```
@@ -704,7 +704,7 @@ a.action > img {
 .summary {
 	color: var(--color-text-maxcontrast);
 	/* add whitespace to bottom of files list to correctly show dropdowns */
-	height: 250px;
+	height: 310px;
```

Maybe we could calculate that value based on the number of items in the dropdown?

---

Good idea but how do we do that?

---

I'd do it inline just with a variable so it would not automatically adapt (which would be ideal but not sure if worth the effort) but at least it would tell more in detail how the height is built:

```suggestion
	$action-menu-items-count: 9;
	height: 44px * ($action-menu-items-count + 0.5);
```

---

+ 0.5 to have some spacing of half an entry at the bottom

---

Thanks! will do it like this :)